web/src/engine/websites/LineWebtoon_e2e.ts

```diff
@@ -0,0 +1,26 @@
+import { TestFixture } from '../../../test/WebsitesFixture';
+
```

Comment:
But a template like Keyoapp wouldnt be tested in case you test all websites. If i do that, wouldnt the tests for Linewebtoon websites be done twice?

Reply:
Yes, it would. But realistically running the website tests would always be filtered.
There is no use case to run all in one go due to flakiness and long runtime.